src/contrib/getpot/getpot.h

```diff
@@ -532,7 +532,7 @@ class GetPot
    * multiple threads at once, so we'll wrap access to
    * mutable objects in a mutex.
    */
-  GETPOT_MUTEX_DECLARE;
+  GETPOT_MUTEX_DECLARE
 
```

Comment:
Okay, I get it now.  Stray semicolons are fine for -Wpedantic (and would be easy to work around if they weren't) in function/method bodies, but not in struct layouts.

And I can't think of any workaround that avoids the warning and keeps the same syntax but doesn't pointlessly pad the struct in the !USE_LIBMESH case, so I guess this is fine.